src/TableTransforms.jl

```diff
@@ -11,7 +11,8 @@ using Transducers: tcollect
 using LinearAlgebra
 using Statistics
 using PrettyTables
-
+using CategoricalArrays
+import Base.@kwdef
```

Comment:
Please add `using CategoricalArrays` in the block containing the other using statements, and drop the `import Base.@kwdef`. It is easy enough to write the struct outer constructor with keyword args manually, and we avoid adding a dependency on this macro.